src/Workspaces/Core/Portable/Execution/AbstractReferenceSerializationService.cs

```diff
@@ -744,6 +744,11 @@ public IEnumerable<ITemporaryStreamStorage> GetStorages()
             {
                 return _storagesOpt;
             }
+
+            public override string ToString()
```

Comment:
❓ Can we use `[DebuggerDisplay("{Display}")]` instead?

Reply:
Last time I checked, `[DebuggerDisplay]` wasn't respected when evaluating members of a collection, which is the most common pattern for this.

@tmat do you know if that's still the case, or has that been fixed?

Reply:
Works for me:

![image](https://user-images.githubusercontent.com/41759/36041735-5cf841b2-0d7e-11e8-8edb-fe64d011e602.png)